app/components/find/courses/entry_requirements_component/view.rb

```diff
@@ -24,6 +27,14 @@ def degree_grade_content(course)
           degree_grade_hash[course.degree_grade]
         end
 
+        def subject_knowledge_enhancement_content?(course)
```

Comment:
Would this work, it reads slight better I think? 
```
 def subject_knowledge_enhancement_content?(course)
          course.subjects.any? { |subject| subject_id.in  SUBJECT_KNOWLEDGE_ENHANCEMENTS_SUBJECT_CODES }
end
```

Reply:
@syed87 Did you see this feedback from Max?

Reply:
```
def subject_knowledge_enhancement_content?
    course.subjects.any? { |subject| subject.subject_code.in?(SUBJECT_KNOWLEDGE_ENHANCEMENTS_SUBJECT_CODES) }
end

```


Don't think there is a point in passing in the `course`.

Reply:
Yes, I had a chat with Max, this code has changed since the feedback.

`def subject_knowledge_enhancement_content?(course)
          if course.subjects.first.subject_code.nil?
            course.subjects.any? { |subject| SUBJECT_KNOWLEDGE_ENHANCEMENTS_SUBJECT_CODES.include?(subject.subject_code) }
          else
            SUBJECT_KNOWLEDGE_ENHANCEMENTS_SUBJECT_CODES.include?(course.subjects.first.subject_code)
          end
        end`